### DISCUSSION TITLE
Planning Page for Public Meetup

### DISCUSSION THREAD

**Discussion by BrysonMG** [nss-day-cohort-47/C47-Discussions#84] [Topix.random]
# Update
Friday, May 14th at 1:00pm, we are meeting at Centennial Park in Nashville. This is a potluck, so feel free to bring something for lunch. Looking forward to seeing everyone!

**Comment by bvinson23** [comment]
The new 5th and Broadway place has a food hall if we wanted to meet up for lunch or something.

**Comment by MegMcGregor** [comment]
Yay! I could do any day from May 8th-12th, then I'm leaving town.

I'll be fully vaxed, so I could do no masks if we are outside. But happy to wear it either way :)

**Reply by MegMcGregor** [reply]
Also, I live right by centennial park and they FINALLLLY opened the main lawn they've been working on if we wanted to do an outdoor meetup. Would just need to make sure there aren't any events happening (doubtful because ya know, covid.)

**Comment by SidneyCrandall** [comment]
I'll be fully vaccinated, as well. The only day I can't is Wednesday 12. I'm going to see my niece and nephew that day.

**Comment by BenjaminGolden** [comment]
I'm out of town from the 6-12. Anytime on Thursday or Friday should work for me. I have my daughter all day Saturday but I can do Sunday afternoon. 

I think if we are outside masks aren't necessary, but I'll do whatever makes the group feel comfortable.

**Comment by jrossgage** [comment]
Good idea! 
I am open the 11th all day and the 13th until 6pm for a hang. 
I also think if we are outside masks are not necessary, but I have no problem wearing them.

**Comment by bvinson23** [comment]
I'm good for any day during our break. I'm fully vaccinated so I'm cool with whatever the group decides about masks.

**Comment by TarynL** [comment]
Im available the 11th-14th and am also fully vaccinated!

**Comment by MegMcGregor** [comment]
don't worry about scheduling around me! Was just sharing availability in case more people had similar conflicts. If I can be there great, but looks like the days that I'm gone work for the majority of people. If I miss this one I'll be sure to be there for the next meet up :)

**Comment by prestonrobbins** [comment]
ill be gone from the 7th-12th, so if you guys do after that I can swing it, but if not then don't worry about it!

**Comment by srcrank** [comment]
I will be gone from the 10th to the 13th! otherwise am free and flexible!

**Comment by Chadwell-Clark** [comment]
It would be great to see everyone in 3D realtime. I am not convinced that some aren't deep fakes.
I will be very busy next week as my son is graduating from high school and their soccer team is in the regional tournament. I might be able to make a midday Tuesday-Thursday.

**Comment by AutumnFoto** [comment]
Im honestly good with any day and time! I'll be in town all of break, but im also okay with waiting til graduation!

**Reply by AutumnFoto** [reply]
Im also fully vaccinated :)

**Comment by cbjones93** [comment]
I'm not fully vaxxed yet till the 19th, so I might be missing this meetup. Hopefully we can do another one sometime!

**Reply by BrysonMG** [reply]
Being vaccinated isn't required unless you want it to be for yourself. Regardless, we will almost certainly have another meetup after graduation given the interest people have shown to doing so.

**Reply by cbjones93** [reply]
I can be there then, I'll make sure to be masked up! :)
Any day works for me, except sunday.

**Comment by Coachbre** [comment]
I would prefer Tuesday - Thursday since I'd be coming from Knoxville. But I'm open to joining via facetime if everyone chooses another day

**Comment by BenjaminGolden** [comment]
Would it be better to shoot for a weekend, perhaps a Sunday afternoon on either 5/16 or 5/23? It seems like most folks are doing something over our break and It may be easier for us all to get together once everyone has come back to town. Just a thought.

**Reply by MicaelaWyatt** [reply]
I like this idea, it seems like everyone would be back in town by then

**Comment by bvinson23** [comment]
At this point, if anyone wants to grab lunch on Friday the 14th, I'd be down. I think it would be cool to get together even if it's just a smaller group.

**Reply by BrysonMG** [reply]
I just posted on Slack about it
